test/necb/unit_tests/tests/test_necb_activities.rb

```diff
@@ -63,13 +66,88 @@ def test_necb_activities()
+          #
+          #   "space_conditioning_category"
+          #
+          # Unconditioned spaces like attics are expected to be tagged as
+          # "unconditioned". Indirectly-conditioned spaces like plenums are
+          # instead expected to be tagged as "nonresconditioned" (just like the
+          # conditioned spaces they serve).
+          #
+          # Keeping track of which unoccupied spaces are unconditioned matters
+          # greatly for building envelope parameters (ex. construction, thermal
+          # bridging, embodied carbon, costing).
+          model.getSpaces.each do |space|
+            id   = space.nameString
+            zone = space.thermalZone
+            prop = space.additionalProperties.getFeatureAsString(tag)
+
+            err_msg = "BTAP::Activity #{id} empty property (#{cas})?"
+            refute_empty(prop, err_msg)
+            err_msg = "BTAP::Activity #{id} empty zone (#{cas})?"
+            refute_empty(zone, err_msg)
+
+            prop  = prop.get
+            zone  = zone.get
+            id    = zone.nameString
+            tstat = zone.thermostatSetpointDualSetpoint
+
+            err_msg = "BTAP::Activity #{id} empty thermostat (#{cas})?"
+            refute_empty(tstat, err_msg)
+
+            tstat = tstat.get
+            heat  = tstat.heatingSetpointTemperatureSchedule
+            cool  = tstat.coolingSetpointTemperatureSchedule
+            next if space.partofTotalFloorArea
+
+            err_msg = "BTAP::Activity #{id} thermostat, heating (#{cas})?"
+            assert_empty(heat, err_msg)
+            err_msg = "BTAP::Activity #{id} thermostat, cooling (#{cas})?"
+            assert_empty(cool, err_msg)
+
+            # Original OSM identifiers.
+            # ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+            # CASE FullServiceRestaurant (NECB2011) : restaurant (commerce) :
+            #   'attic'              : UNCONDITIONED
+            # CASE LargeOffice (NECB2011) : office (commerce) :
+            #   'GroundFloor_Plenum' : INDIRECTLYCONDITIONED
+            #   'TopFloor_Plenum'    : INDIRECTLYCONDITIONED
+            #   'MidFloor_Plenum'    : INDIRECTLYCONDITIONED
+            # CASE MediumOffice (NECB2011) : office (commerce) :
+            #   'TopFloor_Plenum'    : INDIRECTLYCONDITIONED
+            #   'FirstFloor_Plenum'  : INDIRECTLYCONDITIONED
+            #   'MidFloor_Plenum'    : INDIRECTLYCONDITIONED
+            # CASE QuickServiceRestaurant (NECB2011) : restaurant (commerce) :
+            #   'attic'              : UNCONDITIONED
+            # CASE SmallOffice (NECB2011) : office (commerce) :
+            #   'Attic'              : UNCONDITIONED
+            if attics.include?(building)
+              err_msg = "BTAP::Activity #{id} conditioned (#{cas})?"
+              assert_equal(prop, "unconditioned", err_msg)
+              fdback << "'#{space.nameString}' : UNCONDITIONED"
+            else
```

Comment:
Can't rely on space names for testing attics vs plenums - space names get reset (i.e. "undefined") at some point in the process. Can safely rely on _AdditionalProperties_, though.